bro-spinn/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/brospinn/EksternInntektsmeldingLoeserTest.kt

```diff
@@ -44,7 +44,7 @@ class EksternInntektsmeldingLoeserTest : FunSpec({
     }
     every { spinnKlient.hentEksternInntektsmelding(any()) } returns eksternInntektsmelding
 
-    test("Ved når inntektsmeldingId mangler skal feil publiseres") {
+    xtest("Ved når inntektsmeldingId mangler skal feil publiseres") {
```

Comment:
hvorfor skal testen disables her?

Reply:
Endringen på https://github.com/navikt/helsearbeidsgiver-inntektsmelding/pull/591/files#diff-ab4d85c8acadd187d3db10dc4da0770eeb8912933ec7bf2e0f527a7ee3bb1417L36-R37 gjør at det ikke lenger skal være mulig å lese en melding der inntektsmelding-ID er null (og jeg tror det ikke har vært mulig før heller). Jeg disabler bare for minst mulig diff for øyeblikket.